libp2p/Host.cpp

```diff
@@ -196,6 +196,29 @@ void Host::stopCapabilities()
     }
 }
 
+std::shared_ptr<Peer> Host::peer(NodeID const& _n) const
+{
+    RecursiveGuard l(x_sessions);
+    auto it = m_peers.find(_n);
+    if (it == m_peers.end())
+    {
+        LOG(m_logger) << "Peer " << _n << " not found";
```

Comment:
I think better not to put logging in the simple getter like this (better to log it on the calling side if needed, and you already do in `handshakeFailed`)

Reply:
Good point